src/i18n/translations.ts

```diff
@@ -252,7 +257,10 @@ export default {
         columnTitle: 'Name',
         errorMessage: 'Error',
         errorDescription: 'Could not load the provided Capabilities document, please check the validity of the URL',
-        version: 'Version'
+        invalidUrlErrorMsg: 'Invalid URL specified. Please provide an URL like https://ows.terrestris.de/osm/service',
+        version: 'Version',
+        requestWmsGetCapabilitiesInstruction: 'Please provide any valid GetCapabilities URL und submit ' +
+          'your input with \'Enter\' or click on search button.'
```

Comment:
```suggestion
          'your input by pressing the \'Enter ↲\' key or by clicking on the button right of the field.'
```

Reply:
Thank you very much! I went along with your suggestion.